core-plugins/widgets/Distinct-batchaggregator.json

```diff
@@ -1,15 +1,25 @@
 {
   "metadata": {
-    "spec-version": "1.0"
+    "spec-version": "1.5"
   },
   "configuration-groups": [
     {
       "label": "Distinct",
       "properties": [
         {
-          "widget-type": "textbox",
+          "widget-type": "csv",
           "label": "Fields",
-          "name": "fields"
+          "name": "fields",
+          "widget-attributes": {
+            "delimiter": ",",
+            "value-placeholder": "Field Name"
+          },
+          "plugin-function": {
+            "method": "POST",
```

Comment:
What is the default label for this widget?

Reply:
It shows up as 'Get Schema'